CodeReady Containers/Base.lproj/Main.storyboard

```diff
@@ -39,7 +39,7 @@
                 </application>
                 <menu title="Copy OC Login Command" autoenablesItems="NO" id="aAU-at-WNK">
                     <items>
-                        <menuItem title="Unknow" image="NSStatusNone" enabled="NO" id="5P2-4d-SKo">
+                        <menuItem title="Unknown" image="NSStatusNone" id="5P2-4d-SKo">
```

Comment:
This makes the menuitem clickable, not sure if we want that as there is no action associated with the menuitem?

Reply:
Yes you're right, it's maybe better like this. I dropped the commit.